orca-retrofit2/orca-retrofit2.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/kotlin.gradle"
+
+dependencies {
+  compile project(":orca-core")
+  compile "com.squareup.retrofit2:retrofit:2.4.0"
```

Comment:
Since we already bring in okhttp3, we'll want to exclude retrofit's transitive dep on it so we don't have two versions floating around.